access-mgmt/README.adoc

```diff
@@ -0,0 +1,473 @@
+
+NOTE: The resource identifiers `Starts with` and `Is` support the wildcard character `*` to denote any occurrences of the specified resource. For example, `Topic` = `is *` means any topics in a Kafka instance.
+
+.Supported ACL permissions
+[cols="25%,25%,25%,25%"]
+|===
+h|Resource type
+h|Resource identifier
+h|Access type
+h|Operations
+
+|`Consumer group`
+
+(For consumer group access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Read`
+* `Delete`
+* `Describe`
+
+|`Topic`
+
+(For access to a topic)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Read`
+* `Write`
+* `Create`
+* `Delete`
+* `Alter`
+* `Alter configs`
+* `Describe`
+* `Describe configs`
+
+|`Transactional ID`
+
+(For producer access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Write`
+* `Describe`
+
+|`Kafka instance`
+
+(For access to Kafka instance permissions in ACLs)
+|None
+a|
+* `Allow`
+* `Deny`
+a|
+* `Alter`
+* `Describe`
+|===
+
+[role="_additional-resources"]
+.Additional resources
+https://kafka.apache.org/documentation/#security_authz_primitives[Authorization Primitives^] in Kafka documentation
+
+[id="proc-setting-permissions_{context}"]
+== Setting account permissions in {product}
+
+[role="_abstract"]
+In {product}, you can create Access Control Lists (ACLs) in Kafka instances and set permissions for how user accounts or service accounts can interact with an instance and its resources. Only administrators can manage the level of access of other users or of service accounts created by other users for a Kafka instance. If you're managing only service accounts that you created for a Kafka instance that you created, you don't need administrator rights.
+
+.Prerequisites
+* You're logged in to the {product} web console with administrator access rights. If you're managing only service accounts that you created for a Kafka instance that you created, you don't need administrator rights.
+* You have access to one or more running Kafka instances in {product}.
+* The user account or service account that you're setting permissions for has been created in the organization.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to set permissions for.
+. Click the *Permissions* tab to view the current ACL permissions for this instance.
++
+--
+By default, new Kafka instances contain the following permissions. These permissions allow all accounts in the organization to view topics in the instance, but not to produce or consume messages.
+
+NOTE: The wildcard character `*` denotes any occurrences of the specified resource. For example, `Topic` = `is *` means any topics in a Kafka instance.
+
+.Default ACL permissions for new Kafka instances
+[cols="25%,25%,25%,25%"]
+|===
+h|Account
+h|Resource
+h|Access type
+h|Operation
```

Comment:
This table should also include `Describe` access on the cluster (limited to the describe ACLs API)

Reply:
Done

Reply:
Yes, that looks good.